test/components/util/util.go

```diff
@@ -61,6 +61,8 @@ func KubeconfigPath(t *testing.T) string {
 
 // LokocfgPath gives the LOKOCFG_LOCATION value.
 func LokocfgPath(t *testing.T) string {
+	t.Helper()
```

Comment:
This slipped in into wrong commit, but I guess it's fine :)

Reply:
I'll fix it.